jira/client.py

```diff
@@ -531,6 +531,29 @@ def __init__(
                 for name in f["clauseNames"]:
                     self._fields[name] = f["id"]
 
+    @property
+    def server(self):
+        """Return the server url"""
+        return self._options["server"]
+
+    @property
+    def versioned_rest(self):
+        """Return a versioned rest api url"""
+        return "{server}/{api_path}/{api_version}".format(
+            server=self.server,
+            api_path="rest/"+self._options["rest_path"],
+            api_version=self._options["rest_api_version"]
+        )
+
+    @property
+    def latest_rest(self):
```

Comment:
cleaner, but i need to mention you're moving something private to the public space. 

just need to think about it => more public api => more stuff to maintain :-)

Reply:
Yep moved to a private method based of the existing `_get_url()`